libraries/botframework-streaming-extensions/package.json

```diff
@@ -22,7 +22,7 @@
   "dependencies": {
     "promise.prototype.finally": "^3.1.0",
     "uuid": "^3.3.2",
-    "watershed": "^0.4.0"
```

Comment:
This will probably require changes to BotFrameworkStreamingAdapter to no longer build and pass in a Watershed socket.

Reply:
Good call. Will look into it.